g11n-ws/docs/developer_guide_service.md

```diff
@@ -0,0 +1,27 @@
+## Developer's Guide for Singleton Service
+
+### How to setup Intellij IDEA for development
+
+You can flow below steps to quickly setup your Dev environment:
+
+1. Install JDK 1.8(or later version) and Gradle 4.10.3(or later version)
```

Comment:
Please remove "(or later version)" for both JDK and Gradle.

Reply:
do you mean it doesn't support later version?

Reply:
yes, the compile failed when use JDK 10 or Gradle 5

Reply:
fixed